lib/client.ts

```diff
@@ -878,12 +880,28 @@ export class Client extends EventEmitter {
         Plivo.log.error('Access Token found null. Try to re-login with valid accessToken');
         return false;
       }
-      if (this.getTokenExpiryTimeInEpoch()) {
+
+      let parsedToken = this.parseJwtToken(accessToken);
+
+      if (parsedToken != null && parsedToken['exp'] != null) {
```

Comment:
is `exp` field mandatory while getting a token from getAccessToken()??

Reply:
nope that's not the case. may be i missed that consideration & need to find another way